src/routes/open/components/ReviewInformation/index.tsx

```diff
@@ -31,21 +31,21 @@ const ReviewComponent = ({ userAccount, values }: ReviewComponentProps) => {
   const names = getNamesFrom(values)
   const addresses = getAccountsFrom(values)
   const numOwners = getNumOwnersFrom(values)
+  const { nativeCoin } = getNetworkInfo()
```

Comment:
can you move this outside the component, it's a constant value, that way won't be needed to add it as a useEffect dependency. Check if there are other occurrences of it.